ts/ui/menu/Menu.ts

```diff
@@ -418,18 +418,22 @@ export class Menu {
         this.a11yVar<boolean>('subtitles'),
         this.a11yVar<boolean>('braille'),
         this.a11yVar<boolean>('viewBraille'),
-        this.a11yVar<string>('locale'),
-        this.a11yVar<string> ('speechRules'),
+        this.a11yVar<string>('locale', (value) => SRE.setupEngine({locale: value as string})),
+        this.a11yVar<string> ('speechRules', (value) => {
+          const [domain, style] = value.split('-');
+          this.document.options.sre.domain = domain;
+          this.document.options.sre.style = style;
+        }),
         this.a11yVar<string> ('magnification'),
         this.a11yVar<string> ('magnify'),
         this.a11yVar<boolean>('treeColoring'),
         this.a11yVar<boolean>('infoType'),
         this.a11yVar<boolean>('infoRole'),
         this.a11yVar<boolean>('infoPrefix'),
         this.variable<boolean>('autocollapse'),
-        this.variable<boolean>('collapsible', (collapse: boolean) => this.setCollapsible(collapse)),
-        this.variable<boolean>('inTabOrder', (tab: boolean) => this.setTabOrder(tab)),
-        this.variable<boolean>('assistiveMml', (mml: boolean) => this.setAssistiveMml(mml))
+        this.variable<boolean>('collapsible', (collapse) => this.setCollapsible(collapse)),
+        this.variable<boolean>('inTabOrder', (tab) => this.setTabOrder(tab)),
+        this.variable<boolean>('assistiveMml', (mml) => this.setAssistiveMml(mml))
```

Comment:
There should be no need for explicit lambda expressions. 
But at least we can get rid of parantheses.